### PR TITLE
add(platform): carbon footprint doc

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes
Add documentation on the new carbon footprint tool.

## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [X] I applied the [style guide](../styleguide.md).
- [X] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/concepts/carbon-footprint.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import EarlyBadge from "@site/src/components/Badges/EarlyBadge";
+
+You must be a [super admin](/docs/platform/howto/make-super-admin) to access this feature.
+
+The carbon footprint page shows estimates of the greenhouse gas emissions associated with service usage in your organization.
+
+Emissions are measured in metric tons of carbon dioxide-equivalent, which includes
+multiple greenhouse gasses. The following information is shown on the dashboard:
+
+- Estimated emissions this month: Shows an estimate of emissions for the current calendar
```
Comment (rsalevsky) [inline docs/platform/concepts/carbon-footprint.md:15]: These 3 points are a bit confusing... We show the emissions for the last 3, 6 and 12 months for a specified project on a per month basis. Soon we will also show the total emissions for the selected project and timeframe.

Comment (staceysalamon-aiven) [reply docs/platform/concepts/carbon-footprint.md:15]: There are also the estimated emissions shown at the top of the page. The items in this list reflect the titles in the UI.

File: docs/platform/concepts/carbon-footprint.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Carbon footprint
+early: true
+---
+
+import EarlyBadge from "@site/src/components/Badges/EarlyBadge";
```
Comment (ArthurFlag) [inline docs/platform/concepts/carbon-footprint.md:6]: This import doesn't seem to be used?

```suggestion
```

Comment (staceysalamon-aiven) [reply docs/platform/concepts/carbon-footprint.md:6]: Ah, good catch! I must have changed my mind or something... 😅

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**